src/lib/y2partitioner/dialogs/btrfs_subvolume.rb

```diff
@@ -181,8 +181,7 @@ def content_error
         def uniqueness_error
           return nil unless controller.exist_path?(value)
 
-          # TRANSLATORS: error message, where %s is replaced by a path given by the user.
-          format(_("Subvolume name %s already exists."), value)
+          format(_("A subvolume already exists with this path."), value)
```

Comment:
The sentence is correct but sounds a bit weird. It sounds better ordered like this. "A subvolume with that path already exists". But I would go for something more natural like "There is already a subvolume with that path".